test/extended/cli/apiresources.go

```diff
@@ -41,6 +34,25 @@ var _ = g.Describe("[sig-cli] oc api-resources", func() {
 		o.Expect(out).To(o.ContainSubstring("serviceaccounts"))
 		o.Expect(out).To(o.ContainSubstring("deployments"))
 
+		out, err = oc.Run("api-versions").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("apps/v1"))
+		o.Expect(out).To(o.ContainSubstring("networking.k8s.io/v1"))
+		o.Expect(out).To(o.ContainSubstring("node.k8s.io/v1"))
+		o.Expect(out).To(o.ContainSubstring("storage.k8s.io/v1"))
+	})
+
+	g.It("can output expected information about openshift api-resources [apigroup:build.openshift.io][apigroup:project.openshift.io][apigroup:user.openshift.io]", func() {
```

Comment:
To increase the flexibility it's worth introducing new `g.It` for each apigroup. E.g.:
```go
g.It("can output expected information about openshift build api-resources [apigroup:build.openshift.io]", func() {
   ...
}
g.It("can output expected information about openshift image api-resources [apigroup:image.openshift.io]", func() {
   ...
}
g.It("can output expected information about openshift project api-resources [apigroup:project.openshift.io]", func() {
   ...
}
g.It("can output expected information about openshift user api-resources [apigroup:user.openshift.io]", func() {
   ...
}
```

Reply:
Done